index.js

```diff
@@ -1,4 +1,91 @@
-//cat's are always hungry
-var now = new Date();
-// the cat says
-console.log('Feed the kitties at ' + now);
+window.oranjs = window.oranjs || {};
+
+oranjs.Globals = {
```

Comment:
I'm OK with pulling it in as is for the #hacktoberfest fun, but just suggestion for future refactoring.